cl/citations/tasks.py

```diff
@@ -240,6 +145,11 @@ def find_citations_for_opinion_by_pks(
                 ]
             )
 
+            # Delete the memoized opinion data (not saved in DB)
+            del opinion.source_text
+            del opinion.cleaned_text
+            del opinion.source_is_html
+
```

Comment:
I wouldn't put this in the transaction block, but in any case, seeing this does make me a bit uncomfortable with the design. Is it so much better to pass the opinion object around instead of something else that we could throw away after?

This cleanup is here to fix something a different function did, so it's pretty messy.

Reply:
What if I just removed these lines? I don't think they're actually necessary to have the model save correctly. These memoized fields won't be propagated to the database in any scenario.

Reply:
Yeah, the only risk is if we later create fields with those names. Pretty unlikely.